.github/workflows/ci.yml

```diff
@@ -72,6 +72,13 @@ jobs:
         source .venv/bin/activate
         make check
 
+  shellcheck:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: ShellCheck
+        uses: ludeeus/action-shellcheck@master
+
```

Comment:
If we do the check in `make check`, doesn't that cover checking in CI as well, since that gets called in the lint job?

Reply:
yeah, you're right. should I remove `check-scripts` from `make check`? but I feel like it's a lint job for scripts as well.

Reply:
It makes sense to me to have it as part of `make check`.

Reply:
per offline discussion, we'll keep it separate since editing scripts is much less frequent than editing code